src/Figshare.jl

```diff
@@ -0,0 +1,74 @@
+struct Figshare <: DataRepo
+end
+
+base_url(::Figshare) = "https://api.figshare.com/v2/articles"
+
+function description(repo::Figshare, mainpage)
+    desc = mainpage["description"]
+    authors = [value["full_name"] for value in mainpage["authors"]]
+    author = format_authors(authors)
+    license = mainpage["license"]["name"] * " (" *mainpage["license"]["url"] * ")"
+    date = mainpage["published_date"]
+    paper = mainpage["citation"]
+    
+    escape_multiline_string("""
+    Author: $(author)
+    License: $(license)
+    Date: $(date)
+
+    $(desc)
+
+    Please cite this paper:
+    $(paper)
+    if you use this in your research.
+    """, "\$")
+end
+
+function get_urls(repo::Figshare, page)
+    urls = []
+    for i in page["files"]
+        push!(urls, i["download_url"])
+    end
+    urls
+end
+
+function get_checksums(repo::Figshare, page)
+    checksums = []
+    for i in page["files"]
+        push!(checksums, (:md5, i["computed_md5"]))
+    end
+    checksums
```

Comment:
I think this is better written using a list comprehension.
`[(:md5, ii["computed_md5"]) for ii in page["files"]`

(I prefer `ii` over `i`, as a habit, as you can find it easily with simple search)